src/org/labkey/test/pages/core/admin/LimitActiveUserPage.java

```diff
@@ -0,0 +1,115 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class LimitActiveUserPage extends LabKeyPage<LimitActiveUserPage.ElementCache>
+{
+    public LimitActiveUserPage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
+    }
+
+    public static LimitActiveUserPage beginAt(WebDriverWrapper wrapper)
+    {
+        wrapper.beginAt(WebTestHelper.buildURL("user", "limitActiveUsers"));
+        return new LimitActiveUserPage(wrapper.getDriver());
+    }
+
+    public LimitActiveUserPage userWarning(String value)
+    {
+        elementCache().userWarning.selectByVisibleText(value);
+        return this;
+    }
+
+    public LimitActiveUserPage limitActiveUsers(String value)
+    {
+        elementCache().limitActiveUsers.selectByVisibleText(value);
+        return this;
+    }
```

Comment:
Same comment as above. You could probably even merge these into one method:

```
private void setActiveUsers(Select control, boolean value)
{
    String setValue = value ? "Yes" : "No";
    control.selectByVisibleText(setValue);
}

public LimitActiveUserPage userWarning(boolean value)
{
    setActiveUsers(elementCache().userWarning, value);
    return this;
}

public LimitActiveUserPage limitActiveUsers(boolean value)
{
    setActiveUsers(elementCache().limitActiveUsers, value);
    return this;
}
```

Reply:
changed